Diff pivot readme (#226)
nasa-jpl/open-source-rover: Mechanical/Differential Pivot/README.md

# Differential Pivot

> The Differential Pivot helps to offload forces from one side of the rover to the other while climbing. The differential pivot is crucial in the rocker-bogie suspension system and is designed to help keep all 6 wheels on the ground at all times. It also allows a second attachment point for the body so that it does not free rotate about the center bar that runs through it and connects the two sides of the rocker-bogie.

Reviewer: ```suggestion
The Differential Pivot helps to offload forces from one side of the rover to the other while climbing. The differential pivot is crucial in the rocker-bogie suspension system and is designed to help keep all 6 wheels on the ground at all times. It also allows a second attachment point for the body so that it does not freely rotate about the center bar that runs through it and connects the two sides of the rocker-bogie.
```

> ![Differential Pivot](images/differential_pivot.PNG) 

Reviewer: Would be good to make all images lowercase png (and rename the files as well)

## Features
  * Uses inexpensive RC turnbuckles to give multi-degree rotation about 2 separate points, allowing the geometry to be greatly simplified

## Mechanical Interface/Attachments to Rover:
  * 1/2 inch clamping hub to 1/2 inch aluminum rod on the body
  * 1/2 inch aluminum rod to 1/2 inch clamping hub on rocker-bogie arms

## Machining/Fabrication

1. Aluminum Rods: Cutting

    Parts/Tools Necessary

    | Item | Ref | Qty | Image |
    | :--- | :-- | :-- | :---: |

>     | 0.5" x 3' Aluminum Tube | S16 | 1 | ![S16](/images/components/Structural/S16.png) |

Reviewer: should update these image sizes, example:

![blah](/path/image.png) <!-- .element height="50%" width="50%" -->

    | 0.5" x 4" Aluminum Tube | S18 | 4 | ![S18](/images/components/Structural/S18.png) |
    | Metal Hacksaw or Bandsaw | D4 | 1  | ![D4](/images/components/Tools/D4.png) |
    | Vice clamp or C clamps | D5 | 1 | ![D5](/images/components/Tools/D5.png) |

>     1. Take the 3 foot piece of 0.5 inch aluminum rod S16 and cut it into one 15 inch piece and one 13.5 inch piece. These will now be referred to as the parts S16A and S16B respectively. In addition, take two of the 4 inch aluminum rods S18 and cut them down to 3 inches in length as well.

Reviewer: **bold** the S#'s in the text?

(again, let me know if you want me to fix, I can do this super quickly in sublime text)

    <br/><br/>

    | ![15 inch rod cut](images/15inch_cut.PNG) | ![3 inch rod cut](images/3inch_cut.PNG)|
    |:-:|:-:|
    | Figure 1| Figure 2 |

    <br/><br/>

2. Aluminum Rods: Drilling
    Parts/Tools Necessary
    | Item | Ref | Qty | Image |
    | :--- | :-- | :-- | :---: |
    | 0.5" x 13.5" Aluminum Tube (Modified) | S16B | 1 | ![S16B](/images/components/Structural/S16B.png)|
    | 0.5" x 3" Aluminum Tube (Modified) | S18A | 2 | ![S18A](/images/components/Structural/S18A.png)|
    | Hand Drill or drill press | D3 | 1 |![D3](/images/components/Tools/D3.png)|
    | Drill bit #23 | D6 | 1 | ![D6](/images/components/Tools/D6.jpeg)|
    | Center punch or start drill bit | D7 | 1 |![D7](/images/components/Tools/D7.jpeg) |
    | Vice or V-clamp | D8 | 1 | ![D8](/images/components/Tools/D8.png) | 

    The turnbuckles must be attached to the differential pivot and rocker-bogie arm. We will accomplish this by drilling holes in the aluminum beam S16B and attaching two 5-hole aluminum bars on each side.
    

>     1. Using a vice or clamp, firmly grab onto the 0.5 x 13.5 inch rod S16B with the end extending out around 2 inches from the edge of the vice/clamp. Mark the dimensions as shown in Figure 3. Carefully use a center drill to start the a center hole for these holes. It is important that the center hole is as centered as possible to prevent the bit from walk- ing/slipping during drilling, which could result in the bit breaking. Then, use a #23 (0.154 inch diameter) drill bit and drill all the way through both sides of the rod. This makes the S16B’ part.

Reviewer: ```suggestion
    1. Using a vice or clamp, firmly grab onto the 0.5 x 13.5 inch rod S16B with the end extending out around 2 inches from the edge of the vice/clamp. Mark the dimensions as shown in Figure 3. Carefully use a center drill to start the a center hole for these holes. It is important that the center hole is as centered as possible to prevent the bit from walking/slipping during drilling, which could result in the bit breaking. Then, use a #23 (0.154 inch diameter) drill bit and drill all the way through both sides of the rod. This makes the S16B’ part.
```

    <br/><br/>

    | ![differential_pivot_drilling](images/differential_pivot_cut.PNG) |
    | :--: |
    | Figure 3 |

    <br/><br/>

>     2. Test the holes by taking the 5 Hole Aluminum Beams S21 and screws B7 and making sure that the screws go all the way through as shown in Figure 4. If they do not fit, you can Figure 3: Testing the differential pivot holes Flip the rod around and repeat the steps for the other side, making sure to align the holes’ axes as much as possible such that the holes are all parallel to the previous set.

Reviewer: ```suggestion
    2. Test the holes by taking the 5 Hole Aluminum Beams S21 and screws B7 and making sure that the screws go all the way through as shown in Figure 4. If they do not fit, you can file/drill the hole out until they do pass all the way through. Flip the rod around and repeat the steps for the other side, making sure to align the holes’ axes as much as possible such that the holes are all parallel to the previous set. (note: the hole size on the Aluminum Beam S21 is halfway between a # 4 and # 6 screw; we used the # 6 screws and just used a hex key to drive the screw through the holes in beam, effectively tapping them as well.)
```

    <br/><br/>

    | ![Test alignment of differential pivot](images/differential_standoff_cut.PNG) |
    | :--: |
    | Figure 5 |

    <br/><br/>

## Mechanical Assembly
  Parts/Tools Necessary

  | Item | Ref | Qty | Image |
  | :--- | :-- | :-- | :---: |
  | Single Pattern Bracket | S8 | 1 | ![S8](/images/components/Structural/S8.jpg) |
  | 0.5" Circular Clamping Hub | S13 | 1 | ![S13](/images/components/Structural/S13.png)|
  | 0.5" x 13.5" Aluminum Tube (Modified) | S16B | 1 | ![S16B](/images/components/Structural/S16B.png)|
  | 0.5" x 3" Aluminum Tube (Modified) | S18B | 2 | ![S18B](/images/components/Structural/S18B.png)|
  | 0.5" Bottom Bore Clamp | S20 | 2 | ![S20](/images/components/Structural/S20.jpg)| 
  | 5 Hole Aluminum Beam | S21 | 8 | ![S21](/images/components/Structural/S21.jpg)|
  | RC Turnbluckle | S32 | 2 | ![S32](/images/components/Structural/S32.jpg)| 
  | #6-32x1/4" Button Head Screw | B1 | 8 | ![B1](/images/components/Screws/B1.png)|

>   | #6-32x1.25" Button Head Screw | B7 | 8 | ![B8](/images/components/Screws/B8.png)|

Reviewer: ```suggestion
  | #6-32x1.25" Button Head Screw | B7 | 8 | ![B8](/images/components/Screws/B7.png)|
```

1. Build clamping hub assembly: Attach the the bottom bore clamping hubs S20 to the single pattern bracket S8 using screws B1. Then mount the 0.5” clamping hub S13 to the bottom of this assembly using B1 screws.

   <br/><br/>
   
  | ![Step1](images/diff_step_1.PNG) | ![Step2](images/diff_step_2.PNG)|
  |:-:|:-:|
  | Figure 6| Figure 7 |

   <br/><br/>

2. Differential Pivot: Attach the turnbuckle S32 to the modified 13.5-inch aluminum rod S16B as shown using the 5-hole aluminum bars S21, washers W2, screws B7 and B9, hex nuts B11 and B12. The outermost screw is the #4 screw, the others are #6.

   <br/><br/>

  | ![Step3](images/diff_step_3.PNG) | ![Step4](images/diff_step_4.PNG)|
  |:-:|:-:|
  | Figure 8 | Figure 9 |

   <br/><br/>

3. Differential Pivot continued: Pass the 13.5inch rod through the clamping hub assembly, making sure to center it as much as possible. Then repeat step 2 for the other side of the differential pivot. If necessary, unscrew the turnbuckles (by twisting the middle) to insert the rods into place, then screw the turnbuckle back together.

   <br/><br/>

  | ![Step5](images/diff_step_5.PNG) | ![Step6](images/diff_step_6.PNG)|
  |:-:|:-:|

>   | Figure 8 | Figure 9 |

Reviewer: ```suggestion
  | Figure 10 | Figure 11 |
```

   <br/><br/>

4. Differential Pivot Vertical rods: Repeat step 2 on each of the the 3 inch aluminum rods S18B. Finally, attach all the pieces of the turnbuckles together. Your differential pivot is now complete.

   <br/><br/>

  | ![Step7](images/differential_pivot.PNG) |
  | :--: |

>   | Figure 10 |

Reviewer: ```suggestion
  | Figure 12 |
```
